package.json

```diff
@@ -35,8 +35,8 @@
     "example": "cd example && ./../node_modules/.bin/babel-node app.js",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "prepublish": "npm run lint && npm run build",
-    "lint": "./node_modules/.bin/eslint ./src",
-    "test": "./node_modules/.bin/babel-node -- ./node_modules/.bin/babel-istanbul cover --report lcov _mocha "
+    "lint": "eslint ./src",
+    "test": "babel-node -- ./node_modules/.bin/babel-istanbul cover --report lcov _mocha "
```

Comment:
I tried that and it breaks the script.  I _think_ this is due to it not been called from an `npm run` command.  My understanding is that here the output from `babel-node` (replaced with `node` in the other PR #7) pipes it's output to Istanbul?